code/game/gamemodes/meteor/meteor.dm

```diff
@@ -18,7 +18,7 @@
 		return
 
 	var/list/wavetype = GLOB.meteors_normal
-	var/meteorminutes = (world.time - SSticker.round_start_time - meteordelay) / 10 / 60
+	var/meteorminutes = (world.time - SSticker.round_start_time - meteordelay) / (10 * 60)
```

Comment:
Why did you leave the math in and not just do `/ 600`?

Reply:
Huh, I thought there was a really good reason why it did `/ 10 / 60` which happens to be unknown to me. But I guess it is a legacy for legacies sake, will do it `/ 600`.

I will also make some general whining at how SS13 code is barely commented. Grumble grumble.